text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,293 @@
+
+    2. Internet Explorer
+
+        - Only Internet Explorer 11 is supported.
+
+- Mobile
+
+    1. All
+
+        - All mobile browsers may work but are not explicitly supported.
+
+Any browsers not listed here may work but are not explicitly supported.
+
+## Proposed policy
+
+In the new support policy, we define two categories of browsers:
+
+1. **Evergreen browsers**, which are browsers that:
+
+   * Support using the latest version on all supported platforms (e.g. you can
+     use the latest Chrome an any supported version of Windows, macOS, Android,
+     etc. The browser is versioned independently from the operating system).
+   * Automatically update whenever a new version is available.
+
+2. **Non-evergreen browsers**, which are any browsers which do not meet the
+   criteria of evergreen browsers. For instance, Safari is a supported browser
+   whose version is tied to the version of macOS and iOS that users use. As
+   users will often wait to update their operating system, many users lag behind
+   on the version of Safari they are using, so it cannot be considered
+   evergreen.
+
+Using these categories, Ember will support the following browsers:
+
+- Evergreen
+  - Desktop
+      1. Google Chrome/Chromium
+      2. Mozilla Firefox
+      3. Microsoft Edge
+  - Mobile
+      1. Google Chrome
+      2. Mozilla Firefox
+
+- Non-evergreen
+  - Desktop
+    1. Safari
+  - Mobile
+    1. Safari
+
+Any browser which is not listed here may work, but is not explicitly supported.
+In addition, in order to recategorize browsers as evergreen or non-evergreen, a
+new RFC must be submitted to update the support policy.
+
+So for instance, if Safari were to begin automatically updating its version
+independently of macOS/iOS versions, and thus fulfill the criteria for evergreen
+browsers, it would still be treated as a non-evergreen browser until this policy
+is updated. This will prevent sudden and unexpected changes in the support
+matrix for Ember users.
+
+### Evergreen browsers
+
+Ember and Ember Data will support **all** versions of evergreen browsers which
+meet any of the following criteria:
+
+1. It is the latest version of the browser.
+2. It is the latest LTS/extended support version of the browser (such as Firefox ESR).
+3. It has at least **0.25%** marketshare usage, based on
+   [statcounter](https://gs.statcounter.com/).
+
+This policy will allow Ember to generally support the most recent 2-3 versions
+of the browser, and account for versions which may become temporary transition
+points. For example, when MS Edge made the switch to the new Chromium-backed
+version, the last non-Chromium version saw higher usage for a slightly longer
+period of time as the update process was not as standard, and required an
+explicit opt-in. These transitionary periods are important, but generally
+speaking temporary, and once the usage drops below the threshold Ember can drop
+support at that point.
```

Comment:
Added a note calling out the support change explicitly. Re: SemVer, I think that this policy effectively captures what we actually do today, rather than changing anything substantial. We already would avoid fixes for specific versions of really old Chrome or Firefox, for instance, or versions that have a bug that is fixed in a newer version (example in FireFox: https://github.com/emberjs/ember.js/issues/19115).

So, if anything, this policy widens our support a bit by saying that we _will_ support browsers for a given time period, especially on LTS versions of Ember, and for LTS versions of browsers.